bundles/org.openhab.binding.modbus.lambda/src/main/resources/OH-INF/thing/heatingcircuit-channel-types.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<thing:thing-descriptions bindingId="modbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
```

Comment:
```suggestion
```

Reply:
Deleted this and some more double empty lines.